README.md

```diff
@@ -7,29 +7,25 @@
+This is a python package for __time explicit Life Cycle Assessment (LCA)__, building on top of the [Brightway](https://docs.brightway.dev/en/latest) LCA framework. 
 
-<p align=center><img width="330" alt="image" src="https://github.com/TimoDiepers/timex/assets/90762029/319089a6-7e16-4aa6-8b68-64e1e9e6f4bb"></p>
+Time explicit LCA aims to treat time more coherently in LCA, by __streamlining dynamic LCI and LCIA with prospective LCI databases__. As such, `timex_lca` enables the consideration of both the __timing of processes and emissions__ (e.g., end-of-life treatment occurs 20 years after construction), as well as the __changing state__ of the production system (e.g., increasing shares of renewable electricity in the future). 
 
-### User inputs:
-Upon creation of the foreground database, the user needs to specify temporally distributed exchanges. `Temporal distributions` can occur at technosphere and biosphere exchanges, can be absolute (e.g. 2024-03-18) or relative (e.g. 3 years before) and can have flexible temporal resolution (years to days). They can also be flexibly spread over time, see `bw_temporalis Temporal distributions`. If no time of occurrence is given, it is assumed that the process or emission occurs at the same time as its consuming or emitting activity. 
+Users can define the timing of process and emission exchanges as *temporal distributions*. `timex_lca` automatically propoagates the temporal information through the supply chain and sources the LCI data from the corresponding time explicit database. As the timing of each emission is recorded, `timex_lca` supports __dynamic LCIA methods__, which are provided for the climate change metrics radiative forcing and global warming potential. 
 
-Second, the user needs to specify which background databases represents what time in a mapping dictionary. A filter function can be applied to exclude a (sub)set of databases from the graph traversal to speed up computation time with large databases.
+Time explicit LCA can provide more representative LCA results for use cases, such as:
+- products with variable or evolving production systems
+- long-lived products
+- products with biogenic carbon
 
-### Computation of time explicit inventory:
-When a `timexLCA` is calculated, it first calculates a static LCA of the system. Using functionalities of the [`bw_temporalis`](https://github.com/brightway-lca/bw_temporalis) package, the static impact information is then used for a priority-first graph traversal, following the largest contributions within the supply chain. During traversal, it uses convolution to propagate the temporal profiles of processes through time. 
+### Quick links:
+All documentation, including detailed description of `timex_lca`: [![Read the Docs](https://img.shields.io/readthedocs/timex?label=documentation)](https://timex.readthedocs.io/en/latest/)
```

Comment:
Instead of the badges just make a list where the text is the link?

Reply:
sure!